source/apiVolontaria/apiVolontaria/tests/tests_factories.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.test import APITransactionTestCase
+
+from ..factories import *
+
+
+class FactoriesTests(APITransactionTestCase):
+
+    def setUp(self):
+        pass
+
+    def test_create_user_object(self):
+        self.obj = UserFactory()
+        self.assertNotEqual(self.obj, None)
+
+    def test_create_admin_object(self):
+        self.obj = AdminFactory()
+        self.assertNotEqual(self.obj, None)
```

Comment:
Actually your tests are good but we want to test more than that.

I think a good start can be :
 - Check if i can populate some part of the object. By forcing value. 
   - ex: i want to create a user with the username `Jack`
   - ex: I want to create an `address` in a specific `country`
 - Check every field of the factory to check if they are populated and if they are in the good format